packages/@react-spectrum/overlays/src/OpenTransition.tsx

```diff
@@ -35,15 +35,15 @@ const OPEN_STATES = {
 export function OpenTransition(
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
   props
-): ReactNode {
+): JSX.Element | ReactElement<any, string | JSXElementConstructor<any>>[] {
```

Comment:
was `any` https://app.unpkg.com/@react-spectrum/overlays@5.7.3/files/dist/types.d.ts#L19 but according to inference, this is more accurate